xml/art_sle_ha_pmremote.xml

```diff
@@ -0,0 +1,428 @@
+    do the following:
+   </para>
+   <procedure>
+    <!--<title>Configuring the Physical Host</title>-->
+    <step>
+     <para>Install your cluster as described in the <citetitle>&instquick;</citetitle>.
+      <!--<xref linkend="art.ha.install.quick"/>-->
+     </para>
+    </step>
+    <step>
+     <para>Make sure you open the TCP port 3121 in your firewall settings.</para>
+     <remark>toms 2017-03-23: what about the ports 2224 and 21064, and UDP port 5405?
+     Should they be opened as well?</remark>
+    </step>
+    <step>
+     <para>
+      Install the packages <package>acemaker-remote</package>
+      and <package>resource-agents</package>.</para>
+    </step>
+    <step>
+     <para>Configure the cluster nodes:</para>
+     <remark>toms 2017-03-23: Should we add all remote nodes into csync.cfg and /etc/hosts ?</remark>
+     <substeps>
+      <step>
+       <para>Allow cluster-related services through the local firewall on all
+        cluster nodes.</para>
+      </step>
+      <step>
+       <para>Create an authentication key to all cluster and remote nodes:</para>
+       <screen>&prompt.root;<command>mkdir</command> -p --mode=0755 /etc/pacemaker
+&prompt.root;<command>dd</command> if=/dev/urandom of=/etc/pacemaker/authkey bs=4k count=1</screen>
+      </step>
+      <step>
+       <para>If you want to enable the <systemitem class="daemon"
+        >pacemaker_remote</systemitem> service, run:</para>
+       <screen>&prompt.root;<command>chkconfig</command> pacemaker_remote on</screen>
+      </step>
+      <step>
+       <para>Restart &pace; with <command>systemctl restart pacemaker</command>.</para>
+      </step>
+      <step>
+       <para>Insert the following line in the <filename>/etc/csync2/csync2.cfg</filename>:</para>
+       <screen>include /etc/pacemaker/authkey</screen>
+      </step>
+      <!-- Run csync2? -->
+     </substeps>
+    </step>
+    <!--<step>
+    <para>Check if all necessary firewall ports for an HA cluster are open
+     (for example, 3121, and more)</para>
+   </step>-->
+    <step>
+     <para>Define a primitive resource per remote node:</para>
+     <screen>&prompt.root;<command>crm</command> configure primitive suse09</screen>
+    </step>
+   </procedure>
+  </sect2>
+
+  <sect2 xml:id="sec.ha.pmremote.config.kvm.guest">
+   <title>Configuring the KVM Guests</title>
+   <para>
+    The following procedure creates a KVM guest node <systemitem
+      class="domainname">&wsII;</systemitem> on your physical host
+    <systemitem class="domainname">&wsI;</systemitem>:
+   </para>
+   <procedure>
+    <step>
+     <para>Create a KVM guest on <systemitem class="domainname"
+      >&wsI;</systemitem>. Make sure to configure the guest with
+     a hostname and a static IP address. </para>
+    </step>
+    <step>
+     <para>Add the KVM guest hostname <systemitem
+      class="domainname">&wsII;</systemitem> and its IP address to the hosts
+     machine at <filename>/etc/hosts</filename>.</para>
```

Comment:
I didn't actually get the phrase "...to the hosts machine..." ;-) But I think you meant the guest hostname needs to be resolvable on cluster nodes, which makes sense.